ui/tests/acceptance/secret-engine-list-view-test.js

```diff
@@ -113,28 +119,60 @@ module('Acceptance | secret-engine list view', function (hooks) {
     await runCmd(deleteEngineCmd('kv'));
   });
 
-  test('enterprise: cannot view list without permissions inside namespace', async function (assert) {
-    this.version = 'enterprise';
-    this.backend = `bk-${this.uid}`;
+  test('enterprise: cannot view list without permissions inside a namespace', async function (assert) {
     this.namespace = `ns-${this.uid}`;
-    await runCmd([`write sys/namespaces/${this.namespace} -force`]);
-    await loginNs(this.namespace, ' ');
+    const enginePath1 = `kv-t1-${this.uid}`;
+    const userDefault = await runCmd(createTokenCmd()); // creates a default user token
```

Comment:
No need to change, but I think it makes a little more sense to say "creates a token with a default policy"